### PR TITLE
housekeeping: sort imports

### PR DESCRIPTION
**Pull request by tony**
Run module and tests through [isort](https://github.com/timothycrosley/isort)

### CODE REVIEW

**Comment by tony** [comment]
@carltongibson rebased

File: django_filters/fields.py [hunk 1]
```diff
@@ -1,17 +1,15 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
```
Comment (carltongibson) [inline django_filters/fields.py]: This kind of combination is OK

File: django_filters/fields.py [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
 
 from django import forms
 from django.utils.dateparse import parse_datetime
-
 from django.utils.encoding import force_str
 from django.utils.translation import ugettext_lazy as _
 
 from .utils import handle_timezone
-from .widgets import RangeWidget, LookupTypeWidget, CSVWidget, BaseCSVWidget
+from .widgets import BaseCSVWidget, CSVWidget, LookupTypeWidget, RangeWidget
```
Comment (carltongibson) [inline django_filters/fields.py]: Again, fine.

File: django_filters/filters.py [hunk 3]
```diff
@@ -16,13 +15,11 @@
 from .compat import pretty_name
 from .conf import settings
 from .constants import EMPTY_VALUES
-from .fields import (
-    Lookup, LookupTypeField, BaseCSVField, BaseRangeField, RangeField,
-    DateRangeField, DateTimeRangeField, TimeRangeField, IsoDateTimeField
-)
+from .fields import (BaseCSVField, BaseRangeField, DateRangeField,
+                     DateTimeRangeField, IsoDateTimeField, Lookup,
+                     LookupTypeField, RangeField, TimeRangeField)
```
Comment (carltongibson) [inline django_filters/filters.py]: Even this is OK, although...

**Comment by tony** [comment]
@carltongibson could you pick one from here? https://isort.readthedocs.io/en/latest/#multi-line-output-modes

File: tests/test_filtering.py [hunk 4]
```diff
@@ -1,49 +1,29 @@
...
 from django_filters.exceptions import FieldLookupError
+from django_filters.filters import (AllValuesFilter, AllValuesMultipleFilter,
+                                    CharFilter, ChoiceFilter,
+                                    DateFromToRangeFilter, DateRangeFilter,
+                                    DateTimeFromToRangeFilter, DurationFilter,
+                                    ModelChoiceFilter,
+                                    ModelMultipleChoiceFilter,
+                                    MultipleChoiceFilter, NumberFilter,
+                                    OrderingFilter, RangeFilter,
+                                    TimeRangeFilter, TypedMultipleChoiceFilter)
```
Comment (carltongibson) [inline tests/test_filtering.py]: These I'd prefer One Per Line, with a single indent.

```
from django_filters.filters import (
    AllValue...,
   etc
)

File: tests/test_filters.py [hunk 5]
```diff
@@ -1,54 +1,31 @@
...
+                                    BaseRangeFilter, BooleanFilter, CharFilter,
+                                    ChoiceFilter, DateFilter,
+                                    DateFromToRangeFilter, DateRangeFilter,
+                                    DateTimeFilter, DateTimeFromToRangeFilter,
+                                    DurationFilter, Filter, ModelChoiceFilter,
+                                    ModelMultipleChoiceFilter,
+                                    MultipleChoiceFilter, NumberFilter,
+                                    NumericRangeFilter, OrderingFilter,
+                                    RangeFilter, TimeFilter, TimeRangeFilter,
+                                    TypedMultipleChoiceFilter, UUIDFilter)
```
Comment (carltongibson) [inline tests/test_filters.py]: This is the one I don't like. 

(One per line with a single indent by preference.)

File: tests/test_filterset.py [hunk 6]
```diff
@@ -1,47 +1,25 @@
...
-from django_filters.filters import BaseRangeFilter
-from django_filters.filters import DateRangeFilter
-from django_filters.filters import FilterMethod
-
+from django_filters.filters import (BaseInFilter, BaseRangeFilter,
+                                    BooleanFilter, CharFilter, ChoiceFilter,
+                                    DateRangeFilter, Filter, FilterMethod,
+                                    ModelChoiceFilter,
+                                    ModelMultipleChoiceFilter, NumberFilter,
+                                    UUIDFilter)
```
Comment (carltongibson) [inline tests/test_filterset.py]: Again, the change is fine but One Per Line with a single indent. 🙂

**Comment by tony** [comment]
@carltongibson Just rebased, how do you like it now?

**Comment by carltongibson** [comment]
Yes. I like it a lot. 😄

File: setup.cfg [hunk 7]
```diff
@@ -3,3 +3,6 @@ license-file = LICENSE
 
 [wheel]
 universal=1
+
+[isort]
+multi_line_output=3
```
Comment (carltongibson) [inline setup.cfg]: Good isort-foo

**Comment by rpkilby** [comment]
Annnnnd @carltongibson broke all the other PR's 😉 

Give me a sec and I'll rebase them.

**Comment by carltongibson** [comment]
Yes. Of course I did.  Oops.

**Pull request merged**